### PR TITLE
argocd-operator release 0.8.0

### PR DESCRIPTION
**Pull request by reginapizza**
**What type of PR is this?**
> /kind chore

**What does this PR do / why we need it**:
Releases argocd-operator 0.8.0
+ also upgrades argocd version to 2.8.4

### CODE REVIEW

File: bundle/metadata/annotations.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: argocd-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.10.0+git
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.31.0
```
Comment (iam-veeramalla) [inline bundle/metadata/annotations.yaml:8]: Please revert this change

File: deploy/olm-catalog/argocd-operator/0.8.0/argocd-operator.v0.8.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -218,8 +218,9 @@ metadata:
     capabilities: Deep Insights
     categories: Integration & Delivery
     certified: "false"
+    createdAt: "2023-10-18T14:33:50Z"
     description: Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes.
-    operators.operatorframework.io/builder: operator-sdk-v1.10.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.31.0
```
Comment (iam-veeramalla) [inline deploy/olm-catalog/argocd-operator/0.8.0/argocd-operator.v0.8.0.clusterserviceversion.yaml:223]: Please revert this change.

**Review by iam-veeramalla — CHANGES_REQUESTED** [review]
Looks good to me otherwise

**Comment by reginapizza** [comment]
ah okay thanks, wasn't sure why that was happening- I think when I switched to my new laptop I just downloaded the latest operator-sdk without thinking about versioning with this. I'll update the PR!

**Comment by reginapizza** [comment]
@iam-veeramalla should I use operator-sdk v.11.0  or v1.10 like it was before here?

**Review by iam-veeramalla — APPROVED** [review]
/lgtm

File: build/util/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# Argo CD v2.8.3
-FROM quay.io/argoproj/argocd@sha256:d40da8f5747415eb7f9b5c2d9b645aecd423888cad9b36e4f986bff8ecf0a786 as argocd
+# Argo CD v2.8.4
+FROM quay.io/argoproj/argocd@sha256:f02d31d4b115d79de3dc45ebeec3a3107b9178d980e69d9dc2b26019d43f3e09 as argocd
```
Comment (jaideepr97) [inline build/util/Dockerfile:2]: please use v2.8.5 instead

**Comment by jaideepr97** [comment]
@reginapizza I would recommend creating an alias for your git commit command so that the signoff is automatically added

**Review by jaideepr97 — APPROVED** [review]
lgtm
thanks @reginapizza

**Pull request merged**